### PR TITLE
Proofreading / remove obsolete function

### PR DESCRIPTION
**Pull request by phkam**

### CODE REVIEW

**Comment by dougdragon** [comment]
@phkam Thanks!

File: index.php [hunk 1]
```diff
@@ -38,10 +38,10 @@ function get_response($key) {
     return $codes_and_values[$key];
 }
 
-function return_key() {
-    global $codes_and_values;
-    return key($codes_and_values);
-}
+# function return_key() {
```
Comment (dougdragon) [inline index.php]: Thanks for catching this! Would you mind removing the function altogether in this PR?

Comment (phkam) [reply index.php]: I removed the function and force-pushed again. Please check if this works for you.

**Comment by dougdragon** [comment]
Looks great, thanks for opening the PR!

**Pull request merged**